src/components/PageHelp.tsx

```diff
@@ -87,6 +87,32 @@ const Help = () => (
             name consists from the repository name plus the package, e.g.
             @ksh/latest/ksh.
         </p>
+        <h3>Via GraphQL</h3>
+        <p>
+            You can construct GraphQL queries and execute them on the dashboard's 
+            backend GraphQL server. The GraphQL backend is usually located at:
+        </p>
+        <p className="padding-left-20">
+            {window.location.origin}
+            /graphql
```

Comment:
This should be a link so that you can go directly to the GraphiQL playground and read the API documentation.

Reply:
Not sure entirely how this works, but after running this via npm and examining the locally hosted page, it does spit out a URL to the GraphiQL playground in a fashion similar to "via URL'.